### PR TITLE
Core team post

### PR DESCRIPTION
**Pull request by rehrar**

### CODE REVIEW

**Comment by ordtrogen** [comment]
Not sure if/how to comment on this. Just read the article, recognised "binaryFate" and remembered he was a guest on mike's podcast. Maybe it would be worth adding a link to the end of the article so that people who read the announcement could learn more about him by listening to the interview:

https://moneromonitor.com/episodes/2017-04-24-Episode-003.html

File: _posts/2018-03-01-core-team-announcement.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
+
```
Comment (anonimal) [inline _posts/2018-03-01-core-team-announcement.md]: >Ultimately, there is not yet a way for a domain name to be owned by a decentralized community

Centralization is a requisite for ownership. One can never "own" something while being decentralized because ownership requires at least one instance of authority. Authority requires dominance which requires followers which, in turn, forms a centralized body. Ergo, multiple authorities equate to multiple points of centralization.

>domain name to be owned by a decentralized community

There is a side-step for the domain issue and it's called kovri/i2p multi-homing:

>>Destinations may be hosted on multiple routers simultaneously, by using the same private and public keys. As both instances will periodically publish their signed LeaseSets to the floodfill peers, the most recently published LeaseSet will be returned to a peer requesting a database lookup. As LeaseSets have (at most) a 10 minute lifetime, should a particular instance go down, the outage will be 10 minutes at most, and generally much less than that. The multihoming function has been verified and is in use by several services on the network.

In other words, you and Nth other people can all "own" the same address after you distribute your site's private keys to said people (not advised unless you trust them or don't mind the potential that they could host something you don't like).

File: _posts/2018-03-01-core-team-announcement.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The responsibilities of the Core Team are as follows (in no particular order):
+    - Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
+    - Manage the codebase of the Monero Project, which includes merging code on Github, keeping backups, and ensuring the safety, security, and free access of the code from any party.
+    - Steward the general donation fund, and spending the Monero there on anything they see fit to further the Monero Project.
+    - Act as trusted signers and distributors of reference clients for the Monero coin, and other related technologies.
+    - Set a direction and vision for the Monero Project
```
Comment (anonimal) [inline _posts/2018-03-01-core-team-announcement.md]: >Set a direction and vision for the Monero Project

This can't be done because the core team is purportedly not an authority. Setting direction and vision is the job of an authority - and centralization is a requisite for authority (see comment above).

File: _posts/2018-03-01-core-team-announcement.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    
+In the event that the community deems the Core Team unworthy or lax in their duties, they are free to fork the project away from the current Core Team, as happened with thankful_for_today.
+
+There are some things that the Core Team are NOT responsible for, despite misconceptions from the community. Some of the more common misconceptions are listed below:
+    - Members of the Core Team are NOT anyone's boss, and no permission is needed from them to do anything.
+    - The Core Team does NOT act as a centralized point of failure, but encourages organic, self-started initiatives that further the ecosystem of Monero.
+    - The Core Team does NOT equal Monero. In the event that one, or all of the Core Team goes rogue, we are to remember that Monero is a movement. A global initiative to further privacy globally, and provide real, fungible, digital money for everyone. This can happen even without the presence of the Core Team.
```
Comment (anonimal) [inline _posts/2018-03-01-core-team-announcement.md]: >Monero is a movement

Monero is a software. Movements are centralized bodies which have leaders and funders - many of whom are working behind the scenes to spread their agenda. Monero is not a movement anymore than Windows is a movement (although Windows is a bowel movement, that's for sure).

>global initiative to further privacy globally

Implying that globalism is a requisite for providing privacy? That would be incorrect.

Monero can scale down to a small, local community and still serve it's purpose. Ironically, globalism is the antithesis of decentralization as it requires being built on centralized structures: both physically, socially, and metaphorically.

File: _posts/2018-03-01-core-team-announcement.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    
+Another requirement that the Core Team has internally however, is that each member be sufficiently active. In the event that someone is not able to be active in Monero, they are expected to step down from their position and the remaining members elect another person to replace them. For the aforementioned reason, tacotime is stepping down effective immediately. Those that were not present for the first few months of Monero will probably not remember tacotime, and will not understand the large role that he played in making Monero what it is today. Together, he and smooth championed the early days of bitmonero, and helped the budding community to navigate the project post-thankfulfortoday. He acted as the lead maintainer of the codebase, a role now filled by fluffypony. It's safe to say that Monero would not be what it is today without the early leadership and work of tacotime, and the members of the Core Team are very thankful for his past dedication, and wish him well for all future endeavors.
```
Comment (anonimal) [inline _posts/2018-03-01-core-team-announcement.md]: Rock on, tacotime.

File: _posts/2018-03-01-core-team-announcement.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+But every change brings new opportunity, and the Core Team is delighted to welcome Jeremie "binaryFate" Dubois-Lacoste on board as the newest member. binaryFate holds a PhD in computer science, with academic expertise in combinatorial systems and optimization algorithms. After working for a couple of years in academic research, he made the decision to become a full-time entrepreneur in the cryptocurrency space. He has been actively investing, trading, and creating businesses in that field since 2012. He has been involved with Monero since May 2014, and has been an active advocate of privacy and fungibility-protecting technologies ever since, both online and in real life
```
Comment (anonimal) [inline _posts/2018-03-01-core-team-announcement.md]: Rock on, @binaryFate.

**Comment by anonimal** [comment]
This PR was merged in less than 3 hours after the commits were made, and it's not an urgent code fix. That's not enough time to allow peer review.

File: _posts/2018-03-01-core-team-announcement.md (hunk 3 above)
Comment (binaryFate) [reply _posts/2018-03-01-core-team-announcement.md]: > Monero is a movement

Monero is a software, but strongly ideals-driven. (Windows is not). Calling it a movement may be a strong word. But going to the other extreme, I would find limiting to define Monero as purely a software.

File: _posts/2018-03-01-core-team-announcement.md (hunk 5 above)
Comment (binaryFate) [reply _posts/2018-03-01-core-team-announcement.md]: Thank you @anonimal :)

**Comment by binaryFate** [comment]
> This PR was merged in less than 3 hours after the commits were made, and it's not an urgent code fix. That's not enough time to allow peer review.

True it is not enough time to review. But in this rare particular case it is clearly posted as a communication from 7 individuals in their own name.
(IMO you do bring very good points in your comments fwiw).

File: _posts/2018-03-01-core-team-announcement.md (hunk 2 above)
Comment (anonimal) [reply _posts/2018-03-01-core-team-announcement.md]: >"Propose a direction and vision for the Monero Project"

I think this is an improvement but the fatal flaw is that this sets the precedent for democracy in a place where votes can be generated by random identities on the fly. Also, politics implies bureaucracy, appointees, officers, authorities etc. so, not only is that dangerous (politicians are bought all the time), but that is also beyond the purview of the core team.

Another danger: since the core team holds FFS funds, if the core team *proposes* an idea, this is essentially a suggestion of direction for any contributor *after* they are funded. If the contributor disagrees with the proposal, and they are outvoted by the community (but said contributor knows better because they have experience and skill-sets of which the community, in this example, doesn't have) the outcome will negatively affect the software and also risk food on the table for the contributor.

>Personally I think it is mostly about helping to keep the early community vision when said community keeps growing.

Referencing https://github.com/monero-project/monero-site/pull/634#discussion_r172493038.

**Comment by iamsmooth** [comment]
> communication from 7 individuals in their own name.

I tend to agree. The only 'corrections' or 'improvements' to a document like this (aside from perhaps clear typo/formatting corrections) should come from those individuals. If someone else disagrees then that can be posted as 'A Response To...'

Even then, once the post has been up a while it doesn't make a lot of sense to retroactively edit it. If the views of the authors have changed they can make a later post accordingly.

**Comment by anonimal** [comment]
>I tend to agree. The only 'corrections' or 'improvements' to a document like this (aside from perhaps clear typo/formatting corrections) should come from those individuals. If someone else disagrees then that can be posted as 'A Response To...'

>Further, once the post has been up a while it doesn't make a lot of sense to retroactively edit it, even by the original authors. If the views of the authors have changed they can make a later post accordingly.

^^^^ **alllllllllll** of the aforementioned logic can be applied to code too. This is a precedent that should **not** be set.

>communication from 7 individuals in their own name.

There is only one author here. If 500,000 people were involved in this post, it wouldn't matter since the only documented fact is that there is a single commit author with no attribution to anyone else. Since the author is not a member of the core team, the precedent has been set that anyone who is not a member of the core team has equal privilege to correct this PR as they see fit.

**Comment by fluffypony** [comment]
rehrar submitted the PR on our instruction and with our approval. The merge was by one of the 7 individuals. It doesn't really matter who submitted the PR. There have been times in the past when dEBRUYNE submitted a PR and I'm listed as the co-author of the post, and that's fine. We have much bigger things to worry about than this;)

**Comment by iamsmooth** [comment]
> alllllllllll of the aforementioned logic can be applied to code too. This is a precedent that should not be set.

That's absurd. You can't fix/improve code by submitting more code claiming to be "A response to"

A blog and a program are fundamentally different items.

> There is only one author here. If 500,000 people were involved in this post, it wouldn't matter since the only documented fact is that there is a single commit author with no attribution to anyone else. Since the author is not a member of the core team, the precedent has been set that anyone who is not a member of the core team
 
Wat?  "author: The Core Team"

> has equal privilege to correct this PR as they see fit.

A "correction" would have to be fixing in some manner what "The Core Team" intended it to convey. I'm not aware of any such correction.

**Comment by anonimal** [comment]
>We have much bigger things to worry about than this

I have a minute to kill.

>This PR was merged in less than 3 hours after the commits were made, and it's not an urgent code fix. That's not enough time to allow peer review.
>rehrar submitted the PR on our instruction and with our approval. The merge was by one of the 7 individuals. It doesn't really matter who submitted the PR.

This justifies that any errors caught in a PR by a community member can and will be ignored at the whim of the maintainers and core team. This is a Monero Project anti-pattern. I advise against doing this.

>That's absurd. You can't fix/improve code by submitting more code claiming to be "A response to"

Yes, you can. It's quite easy: just don't rebase.

>Wat? "author: The Core Team"

There was only one author: @rehrar.

`author: The Core Team` does **not** change the fact that rehrar was the only author. If you want to prove attribution then use git signing-off. The Linux kernel has been doing [this](https://github.com/git/git/blob/v2.8.1/Documentation/SubmittingPatches#L234-L302) for years (in particular, [point C](https://github.com/git/git/blob/v2.8.1/Documentation/SubmittingPatches#L262-L264)).

Responsible git'ing is also one of the reasons why Monero Project contributors and maintainers sign commits. Simply having one core team member approve/merge a patch doesn't imply nor guarantee that all parties involved have agreed to the merge. If you want a 'group document', then make it a group document.

>To "correct" would have to be fixing in some manner what "The Core Team" intended it to convey. I'm not aware of any such correction.

Intention and fact can be entirely different things; thus the reason to allow proper timeframes for collaboration and peer-review. See https://github.com/monero-project/monero-site/pull/634/files.

**Comment by iamsmooth** [comment]
> Intention and fact can be entirely different things

The fact here is that this is a statement from the Core Team. Unless there is a misquotation by @rehrar of what the Core Team has stated, there is nothing to factually correct. 

If you believe that the Core Team has stated something inaccurate, post your own blog post.

**Comment by iamsmooth** [comment]
> Wrong. If you actually took the time to look at https://github.com/monero-project/monero-site/pull/634/files, you would've seen the empirically proven factual inaccuracy regarding website hosting

Again, it doesn't matter. It is factual (and factually accurate) that the Core Team Statement said what it said. 

If the Core Team erred on the matter of web site hosting, anyone is welcome to point out the error or disagree, but in their own statement, not by editing the blog post which is attributed to someone else. To do so would be to _introduce_ misquotation.

Alternately, you are welcome to appeal to the Core Team suggesting that the Core Team post a correction to its errors, but there it will be a judgment call as to whether the error is material enough to justify a correction. @fluffypony seems to think we have more important things to worry about and I would tend to agree, but again you are still welcome to express your disagreement _in your own statement_.

> Everyone is allowed to correct the work PR'd to github

Yes, but the only way to 'correct' an attributed statement is to claim misquotation (though see below).

> Also, there's a C4

By my understanding of C4 (which is from vague recollection admittedly), the PR could be merged and then reverted, creating a permanent log of the inappropriate PR. If you want to insist on that, I don't object.

> But you haven't proven that [this is a statement from the Core Team]

That is a valid point. Indeed the process could be better and perhaps that failing should be noted somehow on the page. A PR adding a disclaimer to the page about the lack of sign off would not be objectionable to me as long as it isn't misleading.
